src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -195,27 +183,14 @@ class Settings extends Component {
     });
 
     // Make API call to the endpoint to delete the device on the server side
-    $.ajax({
-      url:
-        BASE_URL +
-        '/aaa/removeUserDevices.json?macid=' +
-        macid +
-        '&access_token=' +
-        cookies.get('loggedIn'),
-      dataType: 'jsonp',
-      crossDomain: true,
-      timeout: 3000,
-      async: false,
-      success: function(response) {
-        console.log(response);
-      },
-      error: function(errorThrown) {
-        console.log(errorThrown);
-      },
-      complete: function(jqXHR, textStatus) {
+    removeUserDevice({ macid })
+      .then(payload => {})
+      .catch(error => {
+        console.log(error);
+      })
+      .finally(() => {
```

Comment:
Instead of reloading here, we need to implement a logic that the component gets re-rendered. Open up an issue for   it and try to address it in the upcoming PRs.